components/scroll-wrapper/demo/scroll-wrapper.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
```

Comment:
By the way... I took it for a spin in the primary-secondary template, and it seems to work nicely there.  😄 

![image](https://user-images.githubusercontent.com/9042472/116252948-09d99f00-a73e-11eb-94bb-0e23cbd37339.png)